Plugins/Events/Events/ObjectEvents.cpp

```diff
@@ -117,6 +132,43 @@ int32_t AddUseObjectActionHook(CNWSObject *thisPtr, ObjectID oidObjectToUse)
     return retVal;
 }
 
+void OpenInventoryHook(CNWSPlaceable *thisPtr, ObjectID oidOpener)
+{
+
+    auto PushAndSignal = [&](const std::string& ev) -> bool {
+        PushEventData("OBJECT", Utils::ObjectIDToString(thisPtr->m_idSelf));
+        return SignalEvent(ev, oidOpener);
+    };
+
+    bool skipped = false;
+    if (PushAndSignal("NWNX_ON_OBJECT_OPEN_BEFORE"))
+    {
+	s_OpenInventoryHook->CallOriginal<int32_t>(thisPtr, oidOpener);
+    } else {
+	    skipped = true;
+    }
```

Comment:
Please fix the formatting here, four spaces before `s_OpenInventoryHook...`  and put braces on their on line.